web_searchbar_full_width/static/src/xml/web_searchbar_full_width.xml

```diff
@@ -1,15 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates>
-    <t t-extend="ViewManagerAction">
-        <t t-jquery="tr.oe_header_row_top td:nth-child(1)">
-            jQuery(this).attr('colspan', 4);
-        </t>
-        <t t-jquery="tr.oe_header_row_top td:nth-child(2)">
-            var $new_row = jQuery('<tr class="oe_header_row"/>');
-            jQuery(this)
-            .parents('tr')
-            .after($new_row);
-            $new_row.append(jQuery(this).attr('colspan', 4));
+    <t t-extend="ControlPanel">
+        <t t-jquery="div.oe-cp-search-view" t-operation="replace">
```

Comment:
Ok. Thanks for guide, but can we go like this way... As in my latest commit i removed this whole `xml` file. And added `width=100%` style on both `breadcrumb & search-view`.
Revert me, if anything goes wrong... thanks..